libbeat/docs/shared-license-statement.asciidoc

```diff
@@ -0,0 +1,4 @@
+Don't have a license? You can start a 30-day trial. At the end of the trial
```

Comment:
@dedemorton In Kibana, you can start a 30-day trial in Management > License Management.  You might want to mention that in the topic that you add to the Kibana documentation.